tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -780,7 +780,7 @@ public function provide70Cases()
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```

Comment:
Often the multiline comment is used to comment code out temporarly.
As such, we can't indent arbitrary lines; in theory even lines starting with `*` could be something other than a comment, but `*` is an accepted standard, enforced in code and conventions by DocBlocks.

Look at this example;
```php
/*
$string = ' Hello World,
    this is a commented out code,
  | with a line starting with a pipe.
';
 */
$string = 'Hello World!';
```
We could let the user specify his/her favourite "*comment starting char*", but honestly I find it overkill.

I suggest to convert that specific comment to asterisks, and keep the current implementation.